app/src/main/java/com/iot/mobiledevelopment/CustomAdapter.java

```diff
@@ -53,6 +79,18 @@ private CustomViewHolder(final View itemView) {
             poster = itemView.findViewById(R.id.custom_imageView);
             textTitle = itemView.findViewById(R.id.custom_title);
             movieYear = itemView.findViewById(R.id.custom_year);
+
+            itemView.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    if (mOnItemListener != null){
```

Comment:
You can collapse it to: "mOnItemListener != null && getAdaprterPosition != RecyclerView.NO_POSITION"